src/game-engine/application-states/PlayerA/BaseState.ts

```diff
@@ -0,0 +1,13 @@
+import BaseState from '../ApplicationStates';
+import { PLAYER_INDEX, types } from './index';
+
+export default class BasePlayerA extends BaseState {
+  constructor(channel, stake, balances, message=undefined) {
+    super(channel, stake, balances, PLAYER_INDEX, message);
+  }
+
+  get type() {
+    types;
+    return types[this.constructor.name];
```

Comment:
Probably don’t need this if we can use insanceOf

Reply:
This should be next on the chopping block.